KyberAdapter/contracts/KyberAdapter.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "./interfaces/IKyberPool.sol";
+import "./interfaces/IERC20.sol";
+import "./lib/SafeERC20.sol";
+import "./BaseAdapter.sol";
+
+contract KyberAdapter is BaseAdapter {
+    using SafeERC20 for IERC20;
+
+    uint256 public constant PRECISION = 1e18;
+    mapping(address => mapping(address => address)) internal TKNS_TO_POOL;
+
+    constructor(
+        string memory _name,
+        address[] memory _pools,
+        uint256 _swapGasEstimate
+    ) BaseAdapter(_name, _swapGasEstimate) {
+        addPools(_pools);
+    }
+
+    function addPools(address[] memory _pools) public onlyMaintainer {
```

Comment:
Missing NatSpec
Add descriptions for the interface, events and functions
Refer https://docs.soliditylang.org/en/v0.8.17/natspec-format.htm